### DISCUSSION TITLE
Copy as HTML

### DISCUSSION THREAD

**Discussion by bernatagulloesbrina** [TabularEditor/TabularEditor3#253] [Ideas]
For those of us who blog a lot, a copy as HTML straight from tabular editor would be super cool. 
it would avoid a couple of steps for each expression 

it should include measure name, or calc item name, or calculated column name, depending on the case 

no hurry though, I can live with copy-paste for the time being.

**Comment by otykier** [comment]
Are you talking about copying DAX expressions as HTML? I'm hesitant to add that as a feature, since there's a thousand different ways you could represent a DAX expression with HTML...

The DAX editor already preserves the syntax highlighting/formatting when you copy the code into for example Word. So in my opinion, this problem should be solved by whatever tool you use to write your blog (i.e. when you paste formatted text, your blog editor should be able to convert that to HTML). Alternatively, use something like [PrismJS](https://prismjs.com/) on your blog as [described here](https://www.pbimaster.com/syntax-highlight-your-power-query-or-dax-code/) to format your DAX code.

**Comment by bernatagulloesbrina** [comment]
I understand
I'm using a barebones WordPress

I wonder if it would be possible to import the functionality from
daxformatter.com

Oh well, it was worth a shot

**Comment by marcosqlbi** [comment]
We use a plugin for WordPress (SyntaxHighlighter Evolved) to show the formatted code.
You can also get the code in HTML (no need for plugins) or for the WordPress Plugin using https://dax.do/ (look at the "Share" section in the video).

**Comment by otykier** [comment]
Closing this as an issue, as it is well outside the scope of Tabular Editor 3 :-) but feel free to continue the discussion.
